telegram/ext/commandhandler.py

```diff
@@ -301,7 +317,8 @@ def __init__(self,
                  pass_update_queue=False,
                  pass_job_queue=False,
                  pass_user_data=False,
-                 pass_chat_data=False):
+                 pass_chat_data=False,
+                 run_async=False):
 
         self._prefix = list()
         self._command = list()
```

Comment:
@Bibo-Joshi quoting your comment because it is now outdated:
> KeyViews or whatever it's called change, when the dict changes, which is inconvenient here, as we're going to loop over the keys. If we receive an update from an unknown user/chat while doing so, chat_ids will change and we get an error like this one.
The use of list() decouples chat_ids from self.chat_data and just considers the keys present at call-time. The new entry is ignored and will be handled in the next call to update_persistence, which the update from the unknown user triggers.

---

i think it's an excellent explanation, but if you put it is a comment on the code itself, it will help whomever reading the code understand it.